cluster/gce/gci/configure-helper.sh

```diff
@@ -619,6 +619,10 @@ function start-kube-apiserver {
   if [[ -n "${ENABLE_GARBAGE_COLLECTOR:-}" ]]; then
     params+=" --enable-garbage-collector=${ENABLE_GARBAGE_COLLECTOR}"
   fi
+  if [[ -n "${NUM_NODES:-}" ]]; then
+    # Heuristic used to compute machine capacity based on number of nodes.
+    params+=" --expected-ram-capacity-mb=$((${NUM_NODES} * 60))"
```

Comment:
@lavalamp - this is some heuristic;
Soon we should start setting up proper requests and limits for our components and we will reuse those here.